torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -7694,12 +7696,14 @@ def _test_verify_model_across_rank(self, use_logger):
 
         @require_backend(DistTestCases.backend_feature["gpu"])
         @require_backends_available(DistTestCases.backend_feature["gpu"])
+        @sandcastle_skip_if(BACKEND == "ucc", "test failing locally with ucc")
```

Comment:
Wondering what error this local failure would manifest to end users? 
Asking to see if it is safe to skip.

Reply:
Just updated the skip message. For both with and without logger versions, it times out after 5 minutes.